namer/consul/src/main/scala/io/buoyant/namer/consul/SvcAddr.scala

```diff
@@ -59,9 +63,9 @@ private[consul] object SvcAddr {
       stats.opens.incr()
       // last good state - if an error occurs, fall back to this if
```

Comment:
I think this comment needs to be updated.  As far as I can tell, `lastGood` is only used for logging purposes now